profile.d/70-systemd-shell-extra.sh

```diff
@@ -12,14 +12,14 @@
 # credentials shell.prompt.prefix, shell.prompt.suffix and shell.welcome, and
 # are propagated into these environment variables by pam_systemd(8).
 
-if [ -n "$SHELL_PROMPT_PREFIX" ]; then
+if [ -n "${SHELL_PROMPT_PREFIX-}" ]; then
```

Comment:
hmm, you are saying this is a bashism in the commit msg. if it is, isn't that a problem, because this stuff is also run by other shells?

Reply:
I have already checked it with both `shellcheck` and `checkbashism` and there's no complaint, so should be ok

Reply:
```
$ cat test
#!/bin/sh

if [ -n "${SHELL_PROMPT_SUFFIX-}" ]; then
    PS1="$PS1$SHELL_PROMPT_SUFFIX"
fi
$ shellcheck test; echo $?
0
$ checkbashisms test; echo $?
0
```

Reply:
indeed, it's a posix thing, found it:

https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/V3_chap02.html

Reply:
Good point. I _think_ this should be portable to other shells (I so far checked with bash, zsh, dash and ksh) but I didn't find an authoritative source which would document that this is portable. Will continue searching and update here accordingly.

Reply:
looks good then, but could you update the commit msg, and not reference bash hence (or even better, mention this is a posix shell concept)

Reply:
Thanks! Will update the commit message accordingly.